FitGoal/FitGoal/SuggestedRoutineCell.swift

```diff
@@ -8,10 +8,16 @@
 
 import UIKit
 
-class SuggestedRoutineCell: UICollectionViewCell {
+protocol RoutineDelegate {
```

Comment:
The instance of the delegate inside the cell should be weak. To do that you need to declare the protocol as:

```swift 
protocol RoutineDelegate: AnyObject {
    ...
}
```